src/commcare_cloud/ansible/deploy_riakcs.yml

```diff
@@ -154,7 +154,27 @@
   hosts: riakcs_control_host
   roles:
     - role: riakcs/create_admin_user
-      when: riak_key is undefined or riak_secret is undefined
+#      when: riak_key is undefined or riak_secret is undefined
```

Comment:
Curious why this was removed? The purpose of this is to only create a new user when credentials do not already exist, which should only be on initial cluster setup.

Will this change cause a new user to be created each time this task is run on riakcs hosts?

Reply:
Commented out for debugging, uncommented here: https://github.com/dimagi/commcare-cloud/pull/2038/commits/5edde79f49ec74bfface323491976d3b22b36294